deploy-as-code/charts/core-services/im-services-analytics/values.yaml

```diff
@@ -0,0 +1,60 @@
+# Common Labels
+labels:
+  app: "im-services-analytics"
+  group: "core"
+
+namespace: core-dev
+
+# Ingress Configs
+ingress:
+  enabled: false
+  zuul: true
+  context: "im-services-analytics"
+
+# Container Configs
+image:
+  repository: "im-services-analytics"
+replicas: "1"
+healthChecks:
+  enabled: true
+  livenessProbePath: "/im-services-analytics/health"
+  readinessProbePath: "/im-services-analytics/health"
+appType: "java-spring"
+tracing-enabled: true
+heap: "-Xmx256m -Xms256m"
+memory_limits: 512Mi
+
+# Additional Container Envs
+env: |
+  - name: EGOV_INFRA_INDEXER_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-config
+        key: es-indexer-host
+  {{- if index .Values "global" "serializers-timezone-in-ist" }}
+  - name: SPRING_KAFKA_PRODUCER_VALUE_SERIALIZER
+    value: org.egov.tracer.kafka.serializer.ISTTimeZoneJsonSerializer
+  {{- end }}
+  - name: JAVA_OPTS
+    value: {{ index .Values "heap" | quote }}
+  - name: SERVER_PORT
+    value: "8080"
+  - name: SECURITY_BASIC_ENABLED
+    value: "false"  
+  - name: MANAGEMENT_SECURITY_ENABLED
+    value: "false"
+  - name: EGOV_STATELEVEL_TENANTID
+    valueFrom:
+      configMapKeyRef:
+        name: egov-config
+        key: egov-state-level-tenant-id
+  - name: EGOV_INDEXER_ES_USERNAME
```

Comment:
@varunreddy-egov do we need this?

Reply:
Yea, this is the elasticsearch username.